standard/StandardProject.cmake

```diff
@@ -18,8 +18,14 @@ include(CMakeParseArguments) # Backwards compatibility
 
 # This must be a macro since project defines scope-local variables
 # that we generally rely on being in the root context.
+# This make use of a little-known feature that allows us to overload builtin functions
+# IMPORTAINT: Note that if there is no *direct* call to 'project' in the root CMakeLists.txt
+# file, CMake will automatically inject one with the default languages of C and CXX.
```

Comment:
We found this to be not the case, at least in my local tests, so this comment appears stale.